src/app/utils/__tests__/date.spec.ts

```diff
@@ -67,4 +71,32 @@ describe('Date Util', () => {
       })
     })
   })
+
+  describe('getStartOfDayInSingaporeTime', () => {
```

Comment:
This test set could benefit from more test cases for branch coverage! So like in the cases of:
- If no options are passed in
- If only date is passed in
- If only timezone is passed in

Reply:
ahh, i missed it after added the param, added in the new commit, thanks